### PR TITLE
adding schools from Goettingen and NLSchB

### PR DESCRIPTION
**Pull request by livgm**

### CODE REVIEW

File: companies/igs-goe.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "slug": "igs-goe",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "public body",
+        "school"
+    ],
+    "name": "Georg-Christoph-Lichtenberg-Gesamtschule Göttingen-Geismar",
+    "address": "Schulweg 22\n37083 Göttingen",
```
Comment (baltpeter) [inline companies/igs-goe.json]: Please add the country (`Deutschland`) to the address.

File: companies/nlschb.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    "slug": "nlschb",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "public body",
+        "school"
+    ],
+    "name": "Niedersächsische Landesschulbehörde",
+    "address": "Auf der Hude 2\n21339 Lüneburg",
```
Comment (baltpeter) [inline companies/nlschb.json]: Same here.

File: companies/thg-goe.json [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+    "slug": "thg-goe",
+    "relevant-countries": [
+        "de"
+    ],
+    "categories": [
+        "public body",
+        "school"
+    ],
+    "name": "Theodor-Heuss Gymnasium Göttingen",
+    "address": "Grotefendstaße 1\n37075 Göttingen",
```
Comment (baltpeter) [inline companies/thg-goe.json]: And here.

File: companies/igs-goe.json (hunk 1 above)
Comment (lmaertens) [reply companies/igs-goe.json]: Done, thanks!

**Comment by baltpeter** [comment]
Thanks again!

**Pull request merged**